src/handlers/hello-world.ts

```diff
@@ -39,6 +39,15 @@ export async function helloWorld(context: Context) {
       issue_number: payload.issue.number,
       body: configurableResponse,
     });
+    if (customStringsUrl) {
+      const response = await fetch(customStringsUrl).then((value) => value.json());
+      await octokit.issues.createComment({
```

Comment:
With this logic, it seems that when `customStringsUrl` is provided, 2 comments get posted (first comes from https://github.com/ubiquibot/plugin-template/pull/20/files#diff-3fcb1026946a1e4b90855060978cdf067206311f0330a9f808bc89c98e3dfb67R36). Is that behavior intended?

Reply:
I think it's fine, it's just for demo purposes

Reply:
Yes, it is.